Packs/Slack/Integrations/SlackEventCollector/SlackEventCollector_test.py

```diff
@@ -200,3 +161,76 @@ def test_test_module(mocker):
 
     mocker.patch.object(Session, 'request', return_value=MockResponse([]))
     assert test_module_command(Client(base_url=''), {}) == 'ok'
+
+
+def test_fetch_events_(mocker):
+    """
+    Given:
+        - fetch-events command execution.
+    When:
+        - The first call "fetch_events_command" was the last run empty, There were 6 results (in 2 pages)
+           assert we returned 4 events according to the limit, and returned the "last_search_stop_point_event_id", "cursor", and next_last_event.
+        - Second, call "fetch_events_command" we have a last run with a cursor and got a response with 3 events (that were already fetched)
+           assert we returned 2 and the last run contains only the "last_fetched_event".
+        - Third, call "fetch_events_command" we got an empty response.
+            assert the last run stayed the same.
+        - The fourth call "fetch_events_command" was in the "last run" the "last_fetched_event", There were 6 results (in 2 pages)
+           assert we returned 4 events according to the limit, and returned the "last_fetched_event" "last_search_stop_point_event_id", "cursor", and next_last_event.
+    """
+    from SlackEventCollector import fetch_events_command, Client
+
+    # first round
+    mock_response1 = MockResponse([
+        {'id': '6', 'date_create': 1521214345},
```

Comment:
Let's make the times to make more sense in terms of order. It could be either in UNIX format or just numbers `1`, `2` etc. But event 1 should have the oldest create time.